### PR TITLE
updating airgap automation to include more downstream cluster tests

### PR DESCRIPTION
**Pull request by slickwarren**
## Issue: 

 https://github.com/rancher/qa-tasks/issues/1396
## Problem

not enough downstream cluster tests in airgap
 
## Solution

add table test format to airgap tests
 
## Testing

waiting on airgap setup to complete to run new jenkins jobs

### CODE REVIEW

**Review by susesgartner — APPROVED** [review]
LGTM

File: tests/v2/validation/provisioning/airgap/README.md [hunk 1]
```diff
@@ -11,17 +11,17 @@ provisioningInput:
   - calico
   # For the kubernetes versions please set it to what is appropriate for that release check
   k3sKubernetesVersion:
-  - v1.23
-  - v1.24
-  - v1.25
+  - v1.30
```
Comment (markusewalker) [inline tests/v2/validation/provisioning/airgap/README.md:14]: Is there a need to have this? I think in our READMEs, constantly updating k8s versions becomes a bit of a pain.

**Review by markusewalker — COMMENTED** [review]
Overall, lgtm. Had a question regarding the readmes, but other than that, will approve once Jenkins runs are provided.

**Review by Josh-Diamond — APPROVED** [review]

File: tests/v2/validation/provisioning/airgap/README.md (hunk 1 above)
Comment (slickwarren) [reply tests/v2/validation/provisioning/airgap/README.md:14]: not necessary to me either, but can we remove in a separate PR?

Comment (markusewalker) [reply tests/v2/validation/provisioning/airgap/README.md:14]: Yeah that's fine, we can do later 👍🏿

**Review by markusewalker — DISMISSED** [review]
lgtm, just needs a rebase

**Review by markusewalker — DISMISSED** [review]

**Review by susesgartner — APPROVED** [review]
LGTM

**Review by markusewalker — COMMENTED** [review]
LGTM, just needs a rebase.

**Pull request closed without merging**